contracts/contracts/Job.sol

```diff
@@ -16,42 +16,50 @@ contract Job is IJob, Ownable {
     // Used for math
     uint256 constant BASE_PERCENTAGE = 10_000;
 
+    // 1% - this is not configurable
+    uint256 constant MINIMUM_SPLIT_CHUNK_PERCENTAGE = 100;
+
     /*************
      * Variables *
      *************/
 
     // 50 paymentTokens ($50)
-    uint256 public MINIMUM_BOUNTY = 50e18;
-    // 50 paymentTokens ($50) - TODO: Should we make this a pct with a hard floor?
-    uint256 public MINIMUM_DEPOSIT = 50e18;
-
-    // 10%
-    uint256 public DAO_FEE = 1000;
-    uint256 constant MAX_DAO_FEE = 2500;
-    // 6%
-    uint256 public RESOLUTION_FEE_PERCENTAGE = 600;
-    uint256 constant MAX_RESOLUTION_FEE_PERCENTAGE = 2500;
-    // 1% - this is not configurable
-    uint256 constant MINIMUM_SPLIT_CHUNK_PERCENTAGE = 100;
-    // Number of seconds after job is completed before job is awarded to engineer
+    uint256 public MINIMUM_BOUNTY = 50e18; // $50
+    // 50 paymentTokens ($50)
+    uint256 public MINIMUM_DEPOSIT = 50e18; // $50
```

Comment:
I'm wondering if we need to start talking about these differently. For example, if we supported Ethereum - these wouldn't be right.

Reply:
Supporting a non-stable coin like ETH would be a significant change to the contract, yes.